css/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+	text-align: center;
+	font-family: 'Maven Pro', sans-serif;
+}
+
+h1 {
+	font-size: 40px;
+	font-weight: bold;
+}
+
+div {
+	width: 350px;
+	height: 114px;
+	margin: auto;
+	}
+
+span{
+	display: inline-block;
+	width: 100px;
+	height: 100px;
+	background-color: mediumturquoise;
+	border: 2px solid black;
+	margin: 2px;
+	border-radius: 10px;
+	overflow: auto;
+	font-size: 80px;
+	font-weight: bold;
+
+}
+
+.blue{
+	background-color: dodgerblue;
```

Comment:
Side note, there's a color called `rebeccapurple`, in memory of Eric Meyer's daughter Rebecca (Eric helped design web standards for CSS).

http://www.economist.com/blogs/babbage/2014/06/digital-remembrance